src/state/state.tsx

```diff
@@ -1,8 +1,44 @@
+import React, { createContext, Dispatch, useContext, useReducer } from 'react';
 import type { ArNSContractState } from '../types';
+import type { Action } from './reducer';
+import testContractState from '../../data/contracts/bLAgYxAdX2Ry-nt6aH2ixgvJXbpsEYm28NgJgyqfs-U.json';
 
 export type GlobalState = {
   arnsSourceContract: ArNSContractState;
   gateway: string;
   connectedWallet: string;
   errors: Array<Error>;
 };
+
+const initialState: GlobalState = {
+  arnsSourceContract: testContractState,
```

Comment:
let's start with 
```suggestion
  arnsSourceContract: { records: {} }
```
and remove the direct import from the local file (we want to use `LocalDataProvider` to fetch this file dynamically)